rocky/reports/views/report_overview.py

```diff
@@ -206,9 +210,16 @@ def run_bulk_actions(self) -> None:
             return self.rerun_reports(report_references)
 
     def delete_reports(self, report_references: list[Reference]) -> None:
+        if not self.organization_member.has_perm("tools.can_delete_oois"):
+            return messages.error(self.request, _("Not enough permissions"))
+
+        for report_reference in report_references:
+            if not issubclass(Reference.from_str(report_reference).class_type, BaseReport):
+                return messages.error(self.request, _("Other OOI type selected than Report"))
```

Comment:
I think that returning the return value of `messages.error`  is confusing, because it suggests that we return something, but we are just passing `None`. I think it is better to make it explicit that we return nothing.

```suggestion
        if not self.organization_member.has_perm("tools.can_delete_oois"):
            messages.error(self.request, _("Not enough permissions"))
            return

        for report_reference in report_references:
            if not issubclass(Reference.from_str(report_reference).class_type, BaseReport):
                messages.error(self.request, _("Other OOI type selected than Report"))
                return
```